moveit_ros/planning/planning_request_adapter_plugins/src/chomp_optimizer_adapter.cpp

```diff
@@ -164,21 +164,23 @@ class CHOMPOptimizerAdapter : public planning_request_adapter::PlanningRequestAd
     return "CHOMP Optimizer";
   }
 
-  virtual bool adaptAndPlan(const PlannerFn& planner, const planning_scene::PlanningSceneConstPtr& planning_scene,
+  virtual bool adaptAndPlan(const PlannerFn& planner, const planning_scene::PlanningSceneConstPtr& ps,
```

Comment:
there's no advantage to this change - longer variable names are easier to read and the style policy of moveit. please revert this.

Reply:
I need the `planning_scene` variable locally and wanted to avoid name duplication. 
Please read the full code before complaining.

Reply:
you could call the new local variable ``planning_scene_diff`` or ``planning_scene_copy``